e2e/terraform/packer/ubuntu-jammy-amd64/cni/cni_args.sh

```diff
@@ -48,7 +48,7 @@ cat <<EOF
         {
           "address": "10.1.0.5/16",
           "gateway": "10.1.0.1",
-          "interface": 1
+          "interface": 0
```

Comment:
this caused the cniArgs test to fail on my laptop (panic in cni lib from index out of range).  not sure how it's been passing in CI, but `0` here works on both.

Reply:
Yeah, the E2E environment has two NICs. A _panic_ in the CNI lib is a little alarming though. Might be worth reporting upstream.